### DISCUSSION TITLE
how can I count member in voice channel?

### DISCUSSION THREAD

**Discussion by haimin22** [Rapptz/discord.py#9033] [Q&A]
Hi, I got a problem.
I want to make a bot that can count member in voice channel.
![image](https://user-images.githubusercontent.com/113004332/198867950-a9db5189-86eb-4dc0-a038-86e791f656c3.png)
Like this image, I want to make my bot print "member in voice channel: 29." every an hour.
Can you help me to make a code?

**Comment by DenverCoder1** [comment]
What have you tried? What did you get stuck on?

All you have to do get the channel and count the length of its [members](https://discordpy.readthedocs.io/en/stable/api.html#discord.VoiceChannel.members)
